README.md

```diff
@@ -1,264 +1,180 @@
+### Cluster Upgrade
 
-### Upgrade Flow
-
-1. Control plane upgrade - Upgrades control plane components i.e API server ,etcd ,kube-scheduler,kubelet etc to target version.
-2. Identification of Managed and Self managed node - The worker nodes are identified as EKS managed and self managed to perform upgrade.
+1. Control plane upgrade - this is handled entirely by AWS once the version upgrade has been requested
+2. Identification of Managed and Self managed node - The worker nodes are identified as EKS managed and self managed to perform upgrade
 3. Managed Node group update - updates managed node group to the specified version
 4. Self Managed Nodegroup update
-   - Launch new nodes with upgraded version and wait until they require ready status for next step.
-   - Mark existing nodes as un-schedulable.
-   - If pod disruption budget (PDB) is present then check for force eviction flag (--force) which is given by user, only then evit the pods or continue with the        flow.
-
-### Objective
-
-1. **To upgrade the cluster in one click** : There are many steps involved in updating a cluster and our main objective is to automate all the steps involved in EKS Cluster Upgradation to one click.
-2. **To reduce manual effort and time** : To update an EKS cluster , customers have to manually perform every step which is quite time taking and also a hectic task so, our objective is to reduce that manual intervention and also to save customer’s time.
-
-### Architecture
-
-This is the architecture of EKS One Click Upgrade:
-<p align="center">
-<img src="./Images/architecture.png" height="600px"/>
-</p>
-
-### Components used
+   - Launch new nodes with upgraded version and wait until they require ready status for next step
+   - Mark existing nodes as unschedulable
+   - If pod disruption budget (PDB) is present then check for force eviction flag (--force) which is given by user, only then evict the pods or continue with the flow
 
-![Technologies Used ](./Images/technologies-used.png)
+## Pre-Requisites
 
-### Workflow
+Before running `eksupgrade`, you will need to have permission for both AWS as well as the Kubernetes cluster itself.
 
-Once the user executes the python script, a pre-flight check is initiated where multiple parameters and versions associated with the cluster are verified. If this check is performed successfully, then the upgrade workflow will be initiated and the cluster along with its components will be upgraded to the target version. After the upgradation process, a final check is done and an email is generated summarizing the details of cluster.
-
-<p align="center">
-<img src="./Images/workflow.png"/>
-</p>
+1. Install `eksupgrade` locally:
 
-### How to use EKS One Click Upgrade via CLI
-
-```zsh
-
-$ aws eks update-kubeconfig --name eks-cluster-name --region aws-region
-$ kubectl edit configmap aws-auth -n kube-system
-
-# Add the IAM user to mapUsers. For example:
```

Comment:
These are somewhat redundant - you can't auth to the cluster without having a role mapping to the cluster already present. We can look at providing the K8s RBAC permissions required later if necessary